src/custom/state/orders/mocks.ts

```diff
@@ -77,7 +76,7 @@ const generateOrder = ({ owner, sellToken, buyToken }: GenerateOrderParams): Ord
     partiallyFillable: false,
     // hacky typing..
     signature: (orderN++).toString().repeat(65 * 2), // 65 bytes encoded as hex without `0x` prefix. v + r + s from the spec
-    receiver: ethers.constants.AddressZero
+    receiver: owner.replace('0x', '')
```

Comment:
Does the API not prefix addresses with `0x`?

Reply:
Hmmm, not sure. I just copied the same come from the line where we set the owner...

@anxolin since you did this in this first place, do you know?